src/apps/components/AppFrame/AppFrame.tsx

```diff
@@ -50,7 +52,7 @@ export const AppFrame: React.FC<Props> = ({
   return (
     <iframe
       ref={frameRef}
-      src={urlJoin(src, `?domain=${shop.domain.host}`)}
+      src={urlJoin(src, `?domain=${shop.domain.host}&id=${appId}`)}
```

Comment:
I know, that's beyond scope of this change, but shouldn't we anticipate `src` value that will already contain query parameters?